src/multiversioning.zig

```diff
@@ -346,6 +364,8 @@ pub const MultiversionHeader = extern struct {
         if (!self.current_flags.visit) return error.InvalidCurrentFlags;
         if (self.current_release == 0) return error.InvalidCurrentRelease;
 
+        stdx.maybe(std.mem.allEqual(u8, &self.reserved, 0));
```

Comment:
TIL `std.mem.allEqual` exists! We also have `stdx.zeroed`.

Reply:
Ah! `stdx.zeroed` is better, because (conveying intention aside) it allows me to unwrap one line in the if :smile:.